app/build.gradle

```diff
@@ -137,10 +135,11 @@ dependencies {
 
     implementation "androidx.fragment:fragment-ktx:1.5.5"
 
-    testImplementation 'org.mockito:mockito-core:3.11.2'
-    testImplementation 'org.mockito:mockito-inline:3.11.2'
+    testImplementation 'org.mockito:mockito-core:5.3.1'
```

Comment:
Remark : this import was removed testImplementation 'org.mockito:mockito-inline:3.11.2' and not updated.

Reply:
`org.mockito:mockito-inline` doesn't work on android.

Reply:
Ok, thank you for this clarification.